package.json

```diff
@@ -129,7 +129,7 @@
     "yup": "^0.29.1"
   },
   "resolutions": {
-    "cypress": "4.11.0"
+    "cypress": "6.5.0"
   },
```

Comment:
This might not be necessary anymore with the new version. Did you check that?

Reply:
Yes, had issues in CI without this.